### DISCUSSION TITLE
v1/en/docs/layouts/docs/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#1014] [General]
# v1/en/docs/layouts/docs/

In this article, we’ll explore the docs layout, which can be used to write project documentation, personal notes, and more.

https://hbs.razonyang.com/v1/en/docs/layouts/docs/

**Comment by liluzibird** [comment]
Hey there again, is there a command or something to add another section for docs? I've tried `hugo new docs/new-section/_index.md` and even `hugo new docs/installation/different-os`. Neither of these have properly formatted index.md. I haven't read anything in the docs that indicate a simpler way to do this, like what you put in your [Write New Articles documentation](https://hbs.razonyang.com/v1/en/docs/content/#write-new-articles). The only way I can see doing it now is to just copy and paste the structure in the docs section already setup manually.

Edit: You can check out the current state of my repo to see what I'm talking about https://github.com/userliluzibert/userliluzibert.github.io

**Reply by razonyang** [reply]
> Hey there again, is there a command or something to add another section for docs? I've tried hugo new docs/new-section/_index.md and even hugo new docs/installation/different-os. Neither of these have properly formatted index.md

Didn't get it. the docs layout was applied `/docs` section by default, in the cases you mentioned above, should work by default without any changes.

**Reply by razonyang** [reply]
If you want to display articles in docs layout on other sections, such as `/notes`, `/guides`, specify the `type` as `docs` on front matter. Take `hugo new notes/_index.md`, `hugo new notes/first/index.md` as examples.

`content/notes/_index.md` as follows.

```
---
type: docs
title: Notes
---
```

`content/notes/first/index.md` as follows.

```
---
type: docs
title: First Note
---
```

**Reply by liluzibird** [reply]
Well, the issue I come across is that when I do `hugo new docs/new-section/_index.md`, I get this format:
```
---
# type : docs
title: {{ replace .Name "-" " " | title }}
date: {{ .Date }}
featured: false
draft: true
comment: true
toc: true
reward: true
pinned: false
carousel: false
series:
categories: []
tags: []
images: []
---

Summary.

<!--more-->

Content.

```

When I want:
```
---
title: {{ replace .Name "-" " " | title }}
linkTitle: {{ replace .Name "-" " " | title }}
linkTitleIcon: <i class="fas fa-fw fa-cog"></i> # The icon of the link title, optional.
navWeight: 1000 # Upper weight gets higher precedence, optional.
---

```

Vice versa for the subdirectory if I wanted to add instructions for a specific operating-system

The best idea I could come up with is to make an archetypes/docs.md with the latter formatting structure. (Shown in my testing branch) However, I couldn't figure out how I would do it for the subdirectory for the operating system.

**Reply by razonyang** [reply]
Archetypes are what you're looking for, read more on https://gohugo.io/content-management/archetypes/.

**Reply by liluzibird** [reply]
Ahh, I see. I should've referred to the Hugo docs. I think the `--kind` feature will help me in creating my `/docs/new-section/operating-system/index.md` issue. Thank you very much
